sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -408,16 +408,19 @@ public void download(OutputStream stream) {
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * {@codesnippet com.azure.storage.blob.specialized.BlobClientBase.downloadToFile#String}
+     * {@codesnippet com.azure.storage.blob.BlobClient.downloadToFile#String}
      *
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob">Azure Docs</a></p>
      *
      * @param filePath A non-null {@link OutputStream} instance where the downloaded data will be written.
+     * @return The properties of the download blob.
      * @throws UncheckedIOException If an I/O error occurs
      */
-    public void downloadToFile(String filePath) {
-        downloadToFile(filePath, null, null, null, null, false, null, Context.NONE);
+    public BlobProperties downloadToFile(String filePath) {
+        downloadToFileWithResponse(filePath, null, null, null, null,
+            false, null, Context.NONE);
+        return getProperties();
```

Comment:
Why are we calling `getProperties` here? Just call `getValue` on the call above.

Reply:
Make changes to the method. My mainline merge messed them a little. Only call the maxOverload now.